tests/webhook_test.go

```diff
@@ -39,55 +87,100 @@ var _ = Describe("Validation webhook", func() {
 		})
 
 		Context("removed existing SSP CR", func() {
+			var (
+				newSsp *ssp.SSP
+			)
+
 			BeforeEach(func() {
 				strategy.SkipSspUpdateTestsIfNeeded()
-			})
 
-			AfterEach(func() {
-				strategy.RevertToOriginalSspCr()
-			})
-
-			It("should fail to create SSP CR with invalid commonTemplates.namespace", func() {
 				foundSsp := getSsp()
-
 				Expect(apiClient.Delete(ctx, foundSsp)).ToNot(HaveOccurred())
 				waitForDeletion(client.ObjectKey{Name: foundSsp.GetName(), Namespace: foundSsp.GetNamespace()}, &ssp.SSP{})
 
 				foundSsp.ObjectMeta = v1.ObjectMeta{
 					Name:      foundSsp.GetName(),
 					Namespace: foundSsp.GetNamespace(),
 				}
-				foundSsp.Spec.CommonTemplates.Namespace = "nonexisting-templates-namespace"
 
-				err := apiClient.Create(ctx, foundSsp)
+				newSsp = foundSsp
+			})
+
+			AfterEach(func() {
+				strategy.RevertToOriginalSspCr()
+			})
+
+			It("should fail to create SSP CR with invalid commonTemplates.namespace", func() {
+				newSsp.Spec.CommonTemplates.Namespace = "nonexisting-templates-namespace"
+
+				err := apiClient.Create(ctx, newSsp)
 				if err == nil {
 					Fail("SSP CR with invalid commonTemplates.namespace created.")
 					return
 				}
 				Expect(err.Error()).To(ContainSubstring(fmt.Sprintf(
 					"creation failed, the configured namespace for common templates does not exist: %v",
-					foundSsp.Spec.CommonTemplates.Namespace,
+					newSsp.Spec.CommonTemplates.Namespace,
 				)))
 			})
+
+			Context("Placement API validation", func() {
+				Context("template-validator", func() {
```

Comment:
This second `Context` is not needed. Each `It` test case already mentions `template-validator`.